src/research.py

```diff
@@ -88,89 +115,311 @@ def get_info(self):
-spear = Weapon('Spear', 40, 20, 1.0, 4, 4, 1.5, 1.5, reload_time=7)
-pike = Weapon('Pike', 80, 25, 1.0, 5, 5, 1.5, 1.5, reload_time=9)
+polehammer = Weapon('Polehammer', 100, 15, 1.0, 8, 2, 2, 1, reload_time=11, armor_pierce=1)
+staff = Weapon('Staff', 20, 7, 1.0, 1, 1, 2, 2, reload_time=7)
+spear = Weapon('Spear', 40, 20, 1.0, 4, 4, 1.5, 1.5, reload_time=7, armor_pierce=1)
+pike = Weapon('Pike', 80, 25, 1.0, 5, 5, 1.5, 1.5, reload_time=9, armor_pierce=1)
 sarissa = Weapon('Sarissa', 160, 35, 1.0, 7, 3, 2, 2, reload_time=12)
 
-all_melee_weapons = [unarmed, polehammer, kopis, mace, falx, club, hammer, dagger, rondel, dirk, shortsword, sword, bastard_sword, claymore, spear, staff, bill, pike, sarissa, axe, flail, morning_star]
-weapon_list = [polehammer, sword, mace, falx, shortsword, bastard_sword, claymore, spear, staff, pike, sarissa, axe, flail, morning_star, bill]
-
-sidearm_list = [dagger, club, mace, kopis, hammer, rondel, dirk, staff, shortsword, axe, spear]
-basic_weapon_list = [club, mace, hammer, staff, shortsword, axe, spear]
-
-sling = Weapon('Sling', 15, 250, 0, 3, 1, 1.8, 1, reload_time=60, ammunition=25, projectile_speed=7)
-javelin = Weapon('Javelin', 100, 125, 0.5, 6, 2, 1.5, 1, reload_time=20, ammunition=3, projectile_speed=5)
+#Long - Kenny Additions
+halberd = Weapon('Halberd', 140, 20, 1.0, 8, 2, 2, 1, reload_time=12, armor_pierce=1)
+poleaxe = Weapon('Poleaxe', 90, 14, 1.0, 6, 2, 2, 1, reload_time=10, armor_pierce=-1)
+
+bardiche = Weapon('Bardiche', 120, 15, 0.8, 10, 1, 1, 0.6, reload_time=11, armor_pierce=1)
+voulge = Weapon('Voulge', 105, 15, 0.9, 8, 1, 1.5, 0.8, reload_time=9)
+naginata = Weapon('Naginata', 220, 14, 2, 4, 0.5, 3, 1.5, reload_time=8, armor_pierce=-1)
+war_spear = Weapon('War Spear', 170, 18, 1.4, 8, 0.3, 0.5, 0.5, reload_time=11)
+sledgehammer = Weapon('Sledgehammer', 150, 14, 0.6, 20, 0.6, 2, 0.3, reload_time=13)
+glaive = Weapon('Glaive', 100, 16, 0.8, 9, 0.6, 1.5, 1.5, reload_time=8)
+lance = Weapon('Lance', 180, 23, 1.5, 9, 1, 2, 1, reload_time=11, armor_pierce=1)
+
+#shields - Kenny Additions
+shield = Weapon('Shield', 10, 3, 1.5, 1, 5, 1, 2, reload_time=10, shield=True)
+leather_shield = Weapon('Leather Shield', 20, 3, 0.5, 1.5, 6, 1, 2, reload_time=10, shield=True)
+wooden_shield = Weapon('Wooden Shield', 35, 3, 0.5, 2, 8, 1, 2, reload_time=10, shield=True)
+
+buckler = Weapon('Buckler', 50, 3, 2, 1.5, 3, 1, 5, reload_time=8, shield=True)
+round_shield = Weapon('Round Shield', 100, 3, 2, 1.5, 10, 1, 4, reload_time=10, shield=True)
+kite_shield = Weapon('Kite Shield', 150, 3, 1.5, 1, 12, 1, 3, reload_time=12, shield=True)
+heater_shield = Weapon('Heater Shield', 70, 3, 1.5, 3, 8, 1, 4, reload_time=10, shield=True)
+board_shield = Weapon('Board Shield', 125, 3, 1, 1, 15, 1, 2.5, reload_time=13, shield=True)
+tower_shield = Weapon('Tower Shield', 150, 3, 1.5, 0.5, 20, 1, 2, reload_time=18, shield=True)
+pavise = Weapon('Pavise Shield', 200, 3, 1.5, 0.75, 25, 1, 2, reload_time=15, shield=True)
```

Comment:
Eventually we will also want to add all of the weapons/shields/equipment you create as well as the mechanics to the README